Robot2018/src/org/usfirst/frc/team199/Robot2018/commands/RunScript.java

```diff
@@ -55,7 +55,7 @@ public RunScript(String scriptName) {
 	    			case "end":
 	    				break outerloop;
 	    			default:
-	    				throw new Exception();
+	    				System.out.println("`" + cmdParts[0] + "`" + " is not a valid command name. Check AAA Reference.");
     			}
```

Comment:
Instead of referencing **cmdParts[0]** throughout this whole loop, wouldn't it make more sense to replace such instances with the variable - **cmdName** - you set to it at the beginning of the loop? Otherwise you don't need the variable, but I think having it be a variable makes more sense because it's less confusing each time it is referenced.